src/os_win.cc

```diff
@@ -326,11 +331,16 @@ char* OS::getenv(const char* variable) {
   wchar_t* variable_w = to_wide_string(variable);
 
   const int BUFFER_SIZE = 32767;
```

Comment:
No +1 here?

Reply:
Not needed.
The `GetEnvironmentVariable` function behaves differently...
The `GetModuleFileNameW` function truncates and returns the number of written characters.
The `GetEnvironmentVariableW` function returns the needed length.